javascript/javascript1/week4/voiceassistant.js

```diff
@@ -0,0 +1,70 @@
+
+let result = "";
+let toDo = [];
+
+function getReply(command) {
+  let commandArray = command.split(" ");
+
+  if (command === "Hello my name is Benjamin") {
+    result = commandArray[commandArray.length - 1];
+    return `Your name is ${result}`;
+  }
+
+  if (command === "What is my name") {
+    return result ? `Your name is ${result}` : "I don't know your name.";
+  }
+
+  if (commandArray[0] === "Add") {
+    const item = commandArray.slice(1, -).join(" ");
+    toDo.push(item);
+    return `${item} added to your to-do.`;
+}
+
+if (commandArray[0] === "Remove") {
+    const item = commandArray.slice(1, -1).join(" ");
+    toDo = toDo.filter(todo => todo !== item);
+    return `${item} removed from your to-do.`;
+}
```

Comment:
"Remove" command doesn't work now. Change "end" property in "slice" to remove items properly.